datasets/polemo2/polemo2.py

```diff
@@ -0,0 +1,131 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""PolEmo2.0 IN and OUT"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{kocon-etal-2019-multi,
+title = "Multi-Level Sentiment Analysis of {P}ol{E}mo 2.0: Extended Corpus of Multi-Domain Consumer Reviews",
+author = "Koco{\'n}, Jan and
+Milkowski, Piotr and
+Za{\'s}ko-Zieli{\'n}ska, Monika",
+booktitle = "Proceedings of the 23rd Conference on Computational Natural Language Learning (CoNLL)",
+month = nov,
+year = "2019",
+address = "Hong Kong, China",
+publisher = "Association for Computational Linguistics",
+url = "https://www.aclweb.org/anthology/K19-1092",
+doi = "10.18653/v1/K19-1092",
+pages = "980--991",
+}
+"""
+
+_DESCRIPTION = """\
+The PolEmo2.0 is a set of online reviews from medicine and hotels domains. The task is to predict the sentiment of a review. There are two separate test sets, to allow for in-domain (medicine and hotels) as well as out-of-domain (products and university) validation.
+"""
+
+_HOMEPAGE = "https://clarin-pl.eu/dspace/handle/11321/710"
+
+_LICENSE = "CC BY-NC-SA 4.0"
+
+_URLs = {
+    "in": "https://klejbenchmark.com/static/data/klej_polemo2.0-in.zip",
+    "out": "https://klejbenchmark.com/static/data/klej_polemo2.0-out.zip",
+}
+
+
+class PolEmo2(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Polemo2(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `polemo2`